custom_components/digitalstrom/translations/pt.json

```diff
@@ -0,0 +1,25 @@
+{
+    "config": {
+        "abort": {
+            "already_configured": "Equipamento já configurado"
+        },
+        "error": {
+            "cannot_connect": "Falha na ligação",
+            "invalid_auth": "Password ou senha errada",
```

Comment:
Thank you for your contribution!
When translated to English this line says "Wrong password or password". Can you confirm this is the correct translation? It should be "Wrong username or password".

Reply:
Sorry my mistake :(